components/mfa/setup/setup.jsx

```diff
@@ -147,9 +146,9 @@ export default class Setup extends React.Component {
     }
 }
 
-Setup.defaultProps = {
-};
 Setup.propTypes = {
```

Comment:
We can move this to the class `static propTypes` attribute. It isn't a stopper for the PR to be merged, of course :)

Reply:
Deferring the `propTypes` reorganization -- for now :)